### ISSUE TITLE
[$15][MSFT-116] Member login page: The links under the 'or login with' and single signon link do not receive tab focus.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2815]
**Steps to Reproduce**
1. Go to Members login:https://accounts.topcoder.com/member. 2. Press the tab key to navigate through the interactive elements on the page. 3. Notice that the links under the section 'or login with' and 'single signon' link do not receive tab focus and are completely skipped.

**Expected Result**
The links should be accessible with the tab key.

**Actual Result**
Not able to access the links such as 'github', under the section 'or login with' with the tab key.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.1.1 Keyboard
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2028

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096269 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096269 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096269 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096269 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@veshu PR https://github.com/appirio-tech/accounts-app/pull/223

**Comment by drasticdpk** [comment]
Hello Tema, 
Here is feedback for this issue.
1. Unable to open link by hitting Enter Key (Whenever get focused on link button. )
2. Extra link has been displayed after tabbing 'Single Sign On' button. 
![image](https://user-images.githubusercontent.com/25297859/61627710-278fef80-aca0-11e9-8083-06b624d2f618.png)
![image](https://user-images.githubusercontent.com/25297859/61627919-a8e78200-aca0-11e9-8c2a-6fe86e45e391.png)

**Comment by lakshmiathreya** [comment]
Please also clarify reg ARIA attributes setting for elements ...

**Comment by mishacucicea** [comment]
The change specific to this issue was just adjusting the tabindex to allow tab navigation
The link in 2 was there all along, check the prod: https://accounts.topcoder.com/member

If that's an issue from an accessibility perspective, we could treat it separately.

Does that make sense, can we accept this issue?

**Comment by mishacucicea** [comment]
@drasticdpk - Are you OK with marking this issue as solved? And treat other accessibility issues that are unrelated to this ticket as separate.

**Comment by drasticdpk** [comment]
Hi @mishacucicea , It would be great if we solved the issue at a same time which is known issue (Caused by current changes). Or you want to create another ticket for above issue please confirm.
cc @veshu @lakshmiathreya

**Comment by mishacucicea** [comment]
Added a new issue to treat that exact problem, let's continue there.
@drasticdpk @lakshmiathreya Can we mark this one as a QA Pass?

**Comment by drasticdpk** [comment]
Thank you @mishacucicea.

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2815 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097268 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097268 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by drasticdpk** [comment]
Windows 
Browser : Chrome /Edge/Firefox

**Comment by cwdcwd** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097268<br/><br/>```This is an automated message for lazybaer via Topcoder X```
